frontend/src/components/explorer/gemBrowser/ReactionTable.vue

```diff
@@ -62,12 +62,12 @@
                 </template>
               </td>
               <td>
-                <template v-for="(RP, i) in r.compartment_str.split(/&#8658;|&#8660;/)">
-                  <template v-if="i !== 0">{{ r.reversible ? ' &#8660; ' : ' &#8658; ' }}</template>
-                  <template v-for="(compo, j) in RP.split(' ,')">
+                <template v-for="(w, i) in r.compartment_str.split(/⇔|⇒/)">
+                  <template v-if="i !== 0">{{ r.reversible ? ' ⇔ ' : ' ⇒ ' }}</template>
```

Comment:
Would it make sense to define `⇔` and `⇒` as constants, since they need to be the same as in the `equationSign` function?

Reply:
Since we don't use global constants elsewhere, I don't feel like it makes sense to add that just for this quite small case

Reply:
We use `messages.js` like global constants (strings), and we use `utils.js` for shared functions. But indeed, there is no dedicated place for this type of constants.